src/config/lang/english.go

```diff
@@ -290,6 +290,10 @@ zarf init --git-push-password={PASSWORD} --git-push-username={USERNAME} --git-ur
 	CmdToolsClearCacheSuccess       = "Successfully cleared the cache from %s"
 	CmdToolsClearCacheFlagCachePath = "Specify the location of the Zarf artifact cache (images and git repositories)"
 
+	CmdToolsDownloadInitShort               = "Download the init package for the current Zarf version into the specified directory."
+	CmdToolsDownloadInitFlagOutputDirectory = "Specify a directory to place the init package in."
```

Comment:
nit, what is our policy on which messages are sentences (end in `.`) and which are not?

Reply:
I made an issue here: https://github.com/defenseunicorns/zarf/issues/1712